packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [3.5.0]
+* Introduce `create_distribution` command for bootstrapping repositories for distribution files and passwords
```

Comment:
What do you think of the following CHANGELOG ?

Introduce the `rx_bloc_cli create_distribution` command, which bootstraps a project containing encryption scripts, certificates and provisioning profiles needed for iOS and Android application distribution. Heavily inspired by the https://codesigning.guide/